src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs

```diff
@@ -590,9 +591,135 @@ public ReadOnlyCollection<string> GetTurnServer()
                 return true;
             };
 
-            NativeWebRTC.ForeachTurnServer(Handle, callback).ThrowIfFailed("Failed to retrieve turn server");
+            NativeWebRTC.ForeachTurnServer(Handle, cb).ThrowIfFailed("Failed to retrieve turn server");
 
             return list.AsReadOnly();
         }
+
+        /// <summary>
+        /// Retrieves the current available statistics information.
+        /// </summary>
+        /// <remarks>The WebRTC must be in the <see cref="WebRTCState.Playing"/></remarks>
+        /// <returns>The WebRTC statistics informations.</returns>
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <exception cref="InvalidOperationException">The WebRTC is not in the valid state.</exception>
+        /// <since_tizen> 10 </since_tizen>
+        public ReadOnlyCollection<WebRTCStatistics> GetStatistics(WebRTCStatisticsCategory category)
```

Comment:
please add a line with `<param name="category">`

BTW, in my opinion the word "available" could be removed from the summary (line 600)

Reply:
Added and removed.